Library/Formula/bullet.rb

```diff
@@ -44,11 +44,20 @@ def install
 
     args << "-DUSE_DOUBLE_PRECISION=ON" if build.with? "double-precision"
 
-    args << "-DBUILD_DEMOS=OFF" if build.without? "demo"
+    with_demo = build.with? "demo"
+    if with_demo && build.with?("shared")
+      # Related to the following warnings when building --with-shared --with-demo
+      # https://gist.github.com/scpeters/6afc44f0cf916b11a226
+      opoo "Installed demos don't work with shared libraries"
+      puts <<-EOS.undent
+      Demos will be disabled due to the use of shared libraries.
+      Static libraries are required for demos installed by homebrew.
```

Comment:
A couple of nits:
* can you indent this text another couple of spaces?
* This isn't a restriction imposed by Homebrew so I think this text is a little confusing; can this read "Demos will be disabled because they link incorrectly against shared libraries"?